bodhi/client/__init__.py

```diff
@@ -79,6 +79,8 @@ def new(user, password, **kwargs):
             print_resp(resp, client)
         except bindings.BodhiClientException as e:
             click.echo(str(e))
+        except Exception as AuthError:
```

Comment:
@cverna Yes, the changes were working, when I tried to do this 
`$ bodhi updates new --user bowlofeggs ejabberd-16.09-5.fc25`
and gave wrong password then it printed "wrong password,please try again"